oauth2client/crypt.py

```diff
@@ -95,7 +97,108 @@ def make_signed_jwt(signer, payload):
     return b'.'.join(segments)
 
 
-def verify_signed_jwt_with_certs(jwt, certs, audience):
+def _verify_signature(message, signature, certs):
```

Comment:
I'm OK with that. I'll change it so that the only caller (`verify_signed_jwt_with_certs`) just sends `certs.values()` instead of `certs`.

Unfortunately, since `verify_signed_jwt_with_certs` is public, we can't change the signature without breaking users (should happen at a `1.x -> 2.x` change, maybe we should pile up things we don't like for that day?).

As for `AppIdentityError` I'm happy to change whatever you'd like, but suggest we punt to another PR. I really dislike the way exceptions are used in this library so am happy to re-work.